app/helpers/course_spreadsheet.rb

```diff
@@ -270,6 +281,9 @@ def create_name_columns(sheet)
         section = reg[Section::types[type]]&.fetch(:section)
         row.push(section&.crn || "<no CRN>")
       end
+      if @course.total_late_days
+        row.push(late_days_remaining(u) || "")
+      end 
       row.push(lecture&.dig(:dropped_date) || "", "", "")
       sheet.push_row(nil, row)
     end
```

Comment:
I wonder if we should add a column for each assignment, splitting the "Lateness" column into "Lateness" and "Lateness penalty"

Reply:
I was wondering that too; It would be great for assignment analytics but is that information that would be used?